R/render.r

```diff
@@ -94,18 +94,43 @@ data_template <- function(pkg = ".", depth = 0L) {
 
 data_open_graph <- function(pkg = ".") {
   pkg <- as_pkgdown(pkg)
-  og <- list()
-  logo <- find_logo(pkg$src_path)
-  if (!is.null(logo)) {
+  og <- pkg$meta$template$opengraph %||% list()
+  og <- keep_supported_open_graph(og)
+  if (is.null(og$image) && !is.null(find_logo(pkg$src_path))) {
+    og$image <- list(src = path_file(find_logo(pkg$src_path)))
+  }
+  if (!is.null(og$image) && !grepl("^http", og$image$src)) {
     site_url <- pkg$meta$url %||% "/"
     if (!grepl("/$", site_url)) {
       site_url <- paste0(site_url, "/")
     }
-    og$image <- paste0(site_url, path_file(logo))
+    og$image$src <- gsub("^man/figures/", "reference/figures/", og$image$src)
```

Comment:
Modifying a path with a regular expression is a bit risky. Do we do this elsewhere in the package?

Reply:
Yes, this line was copied from `tweak_homepage_html()` https://github.com/r-lib/pkgdown/blob/9615409047f102483b7b4ca85c55148f8857d27c/R/html-tweak.R#L244